files/common/usr/bin/unpack-image

```diff
@@ -133,4 +133,8 @@ return=$?
 
 report_progress_inc 100 "Unpacking successful."
 
+if [[ $(systemctl is-active delphix-mgmt.service) == "active" ]]; then
+	su sysadmin -c "system version" >/dev/null
```

Comment:
Why would we want to do this here, as opposed to solving the problem via a change to the app-gate (or qa-gate)?

At first glance, I feel like this is the wrong place to be doing this. As is, this script has no knowledge of the app-gate, and I think it should remain that way. Instead, I'd recommend any users of this script handle updating the app-gate version list on their own, rather than having this script be responsible for that.